core/src/main/java/org/infinispan/context/SingleKeyNonTxInvocationContext.java

```diff
@@ -149,7 +153,7 @@ public void setClassLoader(ClassLoader classLoader) {
 
    @Override
    public boolean hasLockedKey(final Object key) {
-      return isLocked && this.key.equals(key);
+      return isLocked && isKeyEquals(key);
```

Comment:
Hmmm, why? 

Are you just pointing out the choice of method name? Or something else?